lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -44,6 +45,9 @@ export type AuthOptions = {
     clientId: string;
     authority?: Authority;
     knownAuthorities?: Array<string>;
+    instanceMetadata?: Array<IInstanceDiscoveryMetadata>;
+    redirectUri?: string | (() => string);
```

Comment:
`redirectUri` and `postlLogoutRedirectUri` should no longer be part of the client config

Reply:
Done